website/blog/2018-02-26-1.11.0.md

```diff
@@ -0,0 +1,1037 @@
+
+#### Ignore `media` params with SCSS interpolation ([#3801] by [@evilebottnawi])
+
+Prettier was mistakenly breaking code when a media query contained SCSS interpolation. It is now preserved:
+
+<!-- prettier-ignore -->
+```scss
+// Unformatted code:
+$sm-only: '(min-width: 768px) and (max-width: 991px)';
+$lg-and-up: '(min-width: 1200px)';
+
+@media screen and #{$sm-only, $lg-and-up} {
+    color: #000;
+}
+
+// Formatted code (Prettier 1.10.2):
+@media screen and,  {
+	color: #000;
+}
+
+// Formatted code (Prettier 1.11.0):
+$sm-only: "(min-width: 768px) and (max-width: 991px)";
+$lg-and-up: "(min-width: 1200px)";
+
+@media screen and #{$sm-only, $lg-and-up} {
+  color: #000;
+}
+```
+
+#### Don't print two spaces between selector operator and opening bracket ([#3738] by [@evilebottnawi])
+
+Prettier was mistakenly printing an extra space when a CSS selector ended in an operator (a feature supported by SCSS). This has been fixed:
+
+<!-- prettier-ignore -->
+```scss
+// Unformatted code:
+.this > .ends > .with > .an > .operator > {
+  // content
+}
+
+// Formatted code (Prettier 1.10.2):
+.this > .ends > .with > .an > .operator >  {
+  // content
+}
+
+// Formatted code (Prettier 1.11.0):
+.this > .ends > .with > .an > .operator > {
+  // content
+}
+```
+
+#### Don't change case of SCSS function names ([#3768] by [@evilebottnawi])
+
+Prettier was mistakenly lowercasing SCSS function names in certain cases. This has been fixed.
+
+<!-- prettier-ignore -->
+```scss
+// Unformatted code:
+@include breakpoint (getBp(md)) {
+  &:nth-child(2n + 3) {
+    clear: both;
+  }
+}
+
+// Formatted code (Prettier 1.10.2):
+@include breakpoint (getbp(md)) {
+  &:nth-child(2n + 3) {
+    clear: both;
+  }
+}
+
+// Formatted code (Prettier 1.11.0):
+@include breakpoint(getBp(md)) {
+  &:nth-child(2n + 3) {
+    clear: both;
+  }
+}
+```
+
+#### Keep space around Vue deep selector `>>>` ([#3792] by [@evilebottnawi])
+
+Prettier was mistakenly stripping spaces around Vue's deep selector:
+
+<!-- prettier-ignore -->
+```html
+<!-- Unformatted code: -->
+<style scoped>
+  .box >>> .child-component {
+    font-size: 24px;
+  }
+</style>
+
+<!-- Formatted code (Prettier 1.10.2): -->
+<style scoped>
+  .box>>>.child-component {
+    font-size: 24px;
+  }
+</style>
+
+<!-- Formatted code (Prettier 1.11.0): -->
+<style scoped>
+.box >>> .child-component {
+  font-size: 24px;
+}
+</style>
+```
+
+#### Fix `@nest` at-rule ([#3975] by [@evilebottnawi])
+
+Prettier was mistakenly adding spaces around the `&` character in selectors passed to `@nest`, which changed which selector the rule compiled into. This has been fixed:
+
+<!-- prettier-ignore -->
+```scss
+// Unformatted code:
+.title {
+  @nest :global(h1)& {
+    background: red;
+  }
+}
+
+// Formatted code (Prettier 1.10.2):
+.title {
+  @nest :global(h1) & {
+    background: red;
+  }
+}
+
+// Formatted code (Prettier 1.11.0):
+.title {
+  @nest :global(h1)& {
+    background: red;
+  }
+}
+```
+
+### JavaScript
```

Comment:
I'm not sure how granular you want to be with these release notes, but my `define()` tweak I thiiiink made it into this release: https://github.com/prettier/prettier/pull/3830

Reply:
Thanks, I must have missed that one. I'll add an entry for it

Reply:
Thank you, and thanks for all your hard work!